### DISCUSSION TITLE
Google Cloud Query Insights with sqlcommenter on Prisma

### DISCUSSION THREAD

**Discussion by armandoborge** [prisma/prisma#8409] [Q&A] [answered]
I am using Prisma to perform operations over a Google Cloud SQL instance (Postgres) and I've enabled Query Insights in order to monitor CRUD operations. Query Insights provides a way to tag SQL queries using `sqlcommenter`, tagging SQL queries simplifies application troubleshooting, the information provided by these tags in the queries helps to find which application code is causing performance problems.

I didn't found any information about how to integrate it with Prisma, is this supported?
Any help would be appreciated!

More information about Query Insights:
https://cloud.google.com/sql/docs/postgres/query-insights-overview
https://cloud.google.com/sql/docs/postgres/using-query-insights#adding_tags_to_sql_queries

sqlcommenter:
https://google.github.io/sqlcommenter/

**Comment by janpio** [comment, marked as answer]
It is not supported right now, but might be a good feature request. Please open a new feature issue for this, and include pretty much the same information your included here. This would need to be added to the Prisma Client API, then transmitted to the Prisma Query Engine and there added to the query in a similar way than `sqlcommenter` does it.

**Reply by ryands17** [reply]
@armandoborge 👋 

If you haven't already, pls open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) 🙂

**Reply by armandoborge** [reply]
Done!
https://github.com/prisma/prisma/issues/8508

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
